src/main/java/uk/nhs/tis/trainee/notifications/service/NotificationService.java

```diff
@@ -517,9 +554,10 @@ public void saveScheduleHistory(JobDataMap jobDetails, Date when) {
     History.TemplateInfo templateInfo = new History.TemplateInfo(notificationType.getTemplateName(),
         templateVersion.get(), jobDetails.getWrappedMap());
 
-    // Only save when notificationType is correct and in Pilot/Rollout
+    // Only save when notificationType is correct and in Pilot/Rollout. We ignore the completion
+    // status of programme actions since these could change before the notification is sent.
```

Comment:
Note that e.g. a formR could be unsubmitted, so actions can go from completed to incomplete over time.